### PR TITLE
Input an dimage

### PR DESCRIPTION
**Pull request by Asuraking1n**

### CODE REVIEW

File: css/image.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    width: 90%;
+    margin-top: 2rem;
+    border-radius: 10px;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    border: 3px solid #524ea1;
+}
+.img-res{
+    height: 90%;
```
Comment (sakshi006) [inline css/image.css:19]: The image is not proper as it is getting displayed for more than its width. Choose a different image maybe?

**Review by sakshi006 — CHANGES_REQUESTED** [review]
Looks good.

**Comment by TusharJade** [comment]
Looks good to me, just update Iframes code

**Comment by ashifkhn** [comment]
Looks amazing. Good to go i guess.

**Pull request merged**